Backend/Services/obj/project.nuget.cache

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
I think that, that files don't need to be committed because it's cache

Reply:
Fixed